README.md

```diff
@@ -40,7 +40,7 @@ metadata:
   name: example
 spec:
   namespace: istio-system
-  version: v1.22.0
+  version: v1.24.0
```

Comment:
Would be less maintenance long term to remove `version` altogether from the examples. When you create the `Istio` resource, version will get defaulted so it's not necessary to include in the examples.

Reply:
Thats a good idea.